userProfile/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+
+def validate_email_addr(test_email):
+    try:
+        validate_email(test_email)
+    except ValidationError:
+        raise ValidationError('Email is not valid')
+
+
+class userProfile(AbstractUser):
```

Comment:
You don't need to create a new user, you can use the django authentication.
You can use [this](https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html#onetoone) as a reference